apps/prairielearn/src/pages/partials/navbar.ejs

```diff
@@ -20,6 +20,7 @@
 <% } %>
 
 <nav class="navbar navbar-dark bg-dark navbar-expand-md" aria-label="Global navigation">
+  <div class="container-fluid">
```

Comment:
Note that this ends up resulting in _too much_ horizontal spacing in BS4, but IMO that's an acceptable compromise.